server/src/test/java/io/crate/metadata/doc/DocIndexMetadataTest.java

```diff
@@ -99,6 +100,7 @@
 import io.crate.types.StringType;
 
 // @formatter:off
+@Ignore
```

Comment:
leftover?

Reply:
sorry, I should have set it back to draft